packages/fast-development-site-react/src/components/site/menu.tsx

```diff
@@ -12,14 +12,44 @@ export interface ISiteMenuState {
 }
 
 export interface ISiteNavManagedClasses {
+    "@keyframes open-animation": string;
+    "@keyframes close-animation": string;
```

Comment:
these are still animating the waffle in from the right, and the entire box is animating from the center rather than the top left, can you ping @Falkicon and determine if this is expected? it seems disjointed to me.

Reply:
Chatted with Jason again and showed him the other examples. His preference is to stick with what office.com is doing.